ci/build_python.sh

```diff
@@ -31,4 +31,22 @@ rapids-conda-retry mambabuild \
   --channel "${CPP_CHANNEL}" \
   conda/recipes/cuvs
 
+# Build cuvs_bench for each cuda and python version
+rapids-conda-retry mambabuild \
+  --no-test \
+  --channel "${CPP_CHANNEL}" \
+  --channel "${RAPIDS_CONDA_BLD_OUTPUT_DIR}" \
+  conda/recipes/cuvs_bench
+
+# Build cuvs_bench_cpu only in CUDA 11 jobs since it only depends on python
+# version
+RAPIDS_CUDA_MAJOR="${RAPIDS_CUDA_VERSION%%.*}"
+if [[ ${RAPIDS_CUDA_MAJOR} == "11" ]]; then
```

Comment:
Can we do CUDA 12 instead? Someday we will drop CUDA 11 and keeping the newer version makes this easier to migrate.
```suggestion
# Build cuvs_bench_cpu only in CUDA 12 jobs since it only depends on python
# version
RAPIDS_CUDA_MAJOR="${RAPIDS_CUDA_VERSION%%.*}"
if [[ ${RAPIDS_CUDA_MAJOR} == "12" ]]; then
```